### PR TITLE
#49: Changed outdated py to bash script

### PR DESCRIPTION
**Pull request by fbrinker**

### CODE REVIEW

File: docs/user-contributed/swapping-workspaces.html [hunk 1]
```diff
@@ -20,39 +20,28 @@ <h1>User-contributed article: Swapping workspaces</h1>
...
+    fi
+done</tt></pre>
 
 <p>
-A very simple way to use this script is as follows: Put the script in a file,
-named for example switch.py, and put switch.py and i3.py (downloaded from the
-<a href="https://github.com/ziberna/i3-py">Python bindings site</a>) in the
-same folder. I typically put it in $HOME/.i3/, the same directory which
-contains i3's config file. Next, put a keybinding like
+To use this script, i recommend binding it to a keyboard shortcut in your i3 config:
```
Comment (Airblader) [inline docs/user-contributed/swapping-workspaces.html]: i → I

File: docs/user-contributed/swapping-workspaces.html [hunk 2]
```diff
@@ -20,39 +20,28 @@ <h1>User-contributed article: Swapping workspaces</h1>
...
-
-<pre><tt>#!/usr/bin/python2.7
-
-import i3
-# retrieve only active outputs
-outputs = filter(lambda output: output[&#39;active&#39;], i3.get_outputs())
-
-# set current workspace to output 0
-i3.workspace(outputs[0][&#39;current_workspace&#39;])
+<pre><tt>#!/bin/bash
```
Comment (Airblader) [inline docs/user-contributed/swapping-workspaces.html]: #!/usr/bin/env bash

is more portable

**Comment by fbrinker** [comment]
You are right, done :)

**Comment by Airblader** [comment]
Thanks!

**Pull request merged**